Crash module fails to add links

## The Bug
Seems like the parameters of link type and issue key are swapped (tries to link the issue with the other issue key `Duplicate` and complains about link type `MC-12949` not existing. `Duplicate` should be the link type and `MC-12949` the issue key)

## Example Ticket
[MCL-14489](https://bugs.mojang.com/browse/MCL-14489)

## Relevant Log
```
09:34:11.492 	main 	INFO 	Arisa 	104 	[RESPONSE] [MCL-14489] [Crash] Successful
09:34:11.838 	main 	ERROR 	Arisa 	113 	[UPDATE] [MCL-14489] Failed
net.rcarz.jiraclient.JiraException: Failed to link issue MCL-14489 with issue Duplicate
    at net.rcarz.jiraclient.Issue.link(Issue.java:1186)
    at net.rcarz.jiraclient.Issue.link(Issue.java:1124)
    at io.github.mojira.arisa.infrastructure.jira.FunctionsKt$createLink$1$1$1.invokeSuspend(Functions.kt:203)
    at io.github.mojira.arisa.infrastructure.jira.FunctionsKt$createLink$1$1$1.invoke(Functions.kt)
    at arrow.core.Either$Companion.catch(Either.kt:862)
    at arrow.core.Either$Companion.catch(Either.kt:858)
    at io.github.mojira.arisa.infrastructure.jira.FunctionsKt$createLink$1$1.invokeSuspend(Functions.kt:202)
    at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
    at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:56)
    at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:271)
    at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:79)
    at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:54)
    at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source)
    at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:36)
    at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source)
    at io.github.mojira.arisa.infrastructure.jira.FunctionsKt$createLink$1.invoke(Functions.kt:201)
    at io.github.mojira.arisa.infrastructure.jira.FunctionsKt$createLink$1.invoke(Functions.kt)
    at io.github.mojira.arisa.modules.HelpersKt.tryRunAll(Helpers.kt:75)
    at io.github.mojira.arisa.infrastructure.jira.FunctionsKt.applyIssueChanges(Functions.kt:119)
    at io.github.mojira.arisa.ModuleExecutor.executeModule(ModuleExecutor.kt:109)
    at io.github.mojira.arisa.ModuleExecutor.execute(ModuleExecutor.kt:48)
    at io.github.mojira.arisa.ArisaMainKt.main(ArisaMain.kt:67)
    at io.github.mojira.arisa.ArisaMainKt.main(ArisaMain.kt)
Caused by: net.rcarz.jiraclient.RestException: 404 : {"errorMessages":["No issue link type with name 'MC-12949' found.","Issue Does Not Exist"],"errors":{}}
    at net.rcarz.jiraclient.RestClient.request(RestClient.java:171)
    at net.rcarz.jiraclient.RestClient.request(RestClient.java:193)
    at net.rcarz.jiraclient.RestClient.request(RestClient.java:238)
    at net.rcarz.jiraclient.RestClient.post(RestClient.java:328)
    at net.rcarz.jiraclient.RestClient.post(RestClient.java:372)
    at net.rcarz.jiraclient.Issue.link(Issue.java:1184)
     ... 22 common frames omitted 
```

## Expected behavior
Don't fail.

--- a/src/main/kotlin/io/github/mojira/arisa/modules/CrashModule.kt
+++ b/src/main/kotlin/io/github/mojira/arisa/modules/CrashModule.kt
@@ -60,7 +60,7 @@ class CrashModule(
             } else {
                 resolveAsDuplicate()
                 addComment(CommentOptions(dupeMessage, key))
-                createLink(key, "Duplicate")
+                createLink("Duplicate", key)
             }
         }
     }

--- a/src/test/kotlin/io/github/mojira/arisa/modules/CrashModuleTest.kt
+++ b/src/test/kotlin/io/github/mojira/arisa/modules/CrashModuleTest.kt
@@ -674,7 +674,7 @@ class CrashModuleTest : StringSpec({
             priority = NoPriority,
             resolveAsInvalid = { Unit.right() },
             resolveAsDuplicate = { Unit.right() },
-            createLink = { key, type -> type.shouldBe("Duplicate"); key.shouldBe("MC-297").right() },
+            createLink = { type, key -> type.shouldBe("Duplicate"); key.shouldBe("MC-297").right() },
             addComment = { Unit.right() }
         )
 
@@ -789,7 +789,7 @@ class CrashModuleTest : StringSpec({
             priority = NoPriority,
             resolveAsInvalid = { Unit.right() },
             resolveAsDuplicate = { Unit.right() },
-            createLink = { key, type -> type.shouldBe("Duplicate"); key.shouldBe("MC-297").right() },
+            createLink = { type, key -> type.shouldBe("Duplicate"); key.shouldBe("MC-297").right() },
             addComment = { Unit.right() }
         )
 
@@ -828,7 +828,7 @@ class CrashModuleTest : StringSpec({
             priority = NoPriority,
             resolveAsInvalid = { Unit.right() },
             resolveAsDuplicate = { Unit.right() },
-            createLink = { key, type -> type.shouldBe("Duplicate"); key.shouldBe("MC-297").right() },
+            createLink = { type, key -> type.shouldBe("Duplicate"); key.shouldBe("MC-297").right() },
             addComment = { Unit.right() }
         )
 

